com.gitlab.davem.ClamTk.yaml

```diff
@@ -0,0 +1,121 @@
+app-id: com.gitlab.davem.ClamTk
+runtime: org.freedesktop.Platform
+runtime-version: '18.08'
+sdk: org.freedesktop.Sdk
+command: clamtk
+rename-icon: clamtk
+copy-icon: true
+finish-args:
+  - '--filesystem=host'
+  - '--share=ipc'
+  - '--share=network'
+  - '--socket=x11'
+  - '--socket=wayland'
+  - '--filesystem=xdg-run/dconf'
```

Comment:
What the app do with dconf access? if it's only for the app's settings, then those could be removed if migrated to 19.08.

Reply:
It's used for the GTK open folder dialog to have favorites, if that's not wanted though I can drop it.

Reply:
Is the favorites are that important in such case? if not, can you drop the dconf access? Thanks